inference-engine/src/mkldnn_plugin/nodes/broadcast.cpp

```diff
@@ -1,135 +1,186 @@
-                errorMsg.copy(resp->msg, sizeof(resp->msg) - 1);
-            }
-            return PARAMETER_MISMATCH;
-        }
-
-        if (src_dims.size() > dst_dims.size()) {
-            if (resp) {
-                std::string errorMsg = "Output tensor dimension is smaller then input tensor dimension";
-                errorMsg.copy(resp->msg, sizeof(resp->msg) - 1);
-            }
-            return PARAMETER_MISMATCH;
-        }
-
-        InferenceEngine::SizeVector dstStrides = outputs[0]->getTensorDesc().getBlockingDesc().getStrides();
-        InferenceEngine::SizeVector src_aligned(dst_dims.size());
-        InferenceEngine::SizeVector srcStrides_aligned(dst_dims.size());
-        size_t prefix_size = dst_dims.size() - src_dims.size();
-        for (size_t i = 0; i < dst_dims.size(); i++) {
-            if (i < prefix_size) {
-                src_aligned[i] = 1;
-                srcStrides_aligned[i] = srcStrides[0];
-            } else {
-                src_aligned[i] = src_dims[i - prefix_size];
-                srcStrides_aligned[i] = srcStrides[i - prefix_size];
-            }
-        }
-
-        size_t work_amount_dst = dstStrides[0] * dst_dims[0];
-        const uint8_t *src_data = inputs[BROADCAST_INPUT]->cbuffer().as<const uint8_t *>() +
-                                inputs[BROADCAST_INPUT]->getTensorDesc().getBlockingDesc().getOffsetPadding();
-        uint8_t* dst_data = outputs[0]->cbuffer().as<uint8_t *>() +
-                          outputs[0]->getTensorDesc().getBlockingDesc().getOffsetPadding();
-
-        parallel_nt(0, [&](const int ithr, const int nthr) {
-            size_t i, src_idx, start = 0, end = 0;
-            SizeVector counters(dst_dims.size(), 0);
-            splitter(work_amount_dst, nthr, ithr, start, end);
-            for (int j = dst_dims.size() - 1, i = start; j >= 0; j--) {
-                counters[j] = i % dst_dims[j];
-                i /= dst_dims[j];
-            }
-            for (size_t iwork = start * data_size; iwork < end * data_size; iwork += data_size) {
-                for (i = 0, src_idx = 0; i < dst_dims.size(); ++i)
-                    src_idx += counters[i] ? ((counters[i] % src_aligned[i]) * srcStrides_aligned[i]) : 0;
-
-                simple_copy(&dst_data[iwork], data_size, &src_data[src_idx * data_size], data_size);
-
-                for (int j = dst_dims.size() - 1; j >= 0; j--) {
-                    counters[j] = (counters[j] + 1) % dst_dims[j];
-                    if (counters[j] != 0) break;
-                }
-            }
-        });
-
-        return OK;
-    }
-
-private:
-    const size_t BROADCAST_INPUT = 0;
-    const size_t BROADCAST_SHAPE = 1;
-};
-
-REG_FACTORY_FOR(BroadcastImpl, Broadcast);
-
-}  // namespace Cpu
-}  // namespace Extensions
-}  // namespace InferenceEngine
+//// Copyright (C) 2018-2020 Intel Corporation
+//// SPDX-License-Identifier: Apache-2.0
+////
+//
+//#include "base.hpp"
```

Comment:
We should not keep commented code in repository.

Reply:
thanks, removed